packages/cycle-scripts/template/config/JavaScript/flavor.js

```diff
@@ -2,10 +2,6 @@ const dependencies = {
   basics: [
     '@cycle/dom@17.1.0'
   ],
-  language: {
```

Comment:
Are we sure there won't be any specific language dependency that we'll need to install? (I didn't know about typescript to much that's why i left this here)